x/dex/client/cli/query_pool.go

```diff
@@ -0,0 +1,61 @@
+package cli
+
+import (
+	"context"
+	"strconv"
+	"strings"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/duality-labs/duality/x/dex/types"
+	"github.com/spf13/cobra"
+)
+
+func CmdShowPool() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "show-pool [pair-id] [tick-index] [fee]",
+		Short:   "shows a Pool",
+		Example: "show-pool-reserves tokenA<>tokenB [-5] 1",
```

Comment:
```suggestion
		Example: "show-pool-reserves 'tokenA<>tokenB' [-5] 1",
```

As I found out previously the `<>` characters aren't interpreted as part of the string in some shells.

To be more consistent with other CLI commands like https://github.com/duality-labs/duality/blob/v0.3.4/x/incentives/client/cli/tx.go#L113-L118 we could split the `[pair-id]` into `[pairTokenA] [pairTokenB]`

Otherwise just note in the example that the string should be quoted.

Reply:
Good catch

Reply:
Adding quotes to the example. We can fix to make this and the rest of the query commands consistent.